api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,793 @@
+{
```

Comment:
<p>I think using self-documenting constant names is more helpful than overwriting metric_id/region_id/etc. variables all over the notebook and only indicating what they're for in comments. i.e. <code>SALTA_REGION_ID = 10152</code> is better than <code>region_id = 10152  #&nbsp;salta</code></p><p> </p><p>Similarly, I like to use</p><p> </p><p><code>from api.client.lib import REGION_LEVELS</code></p><p><code>gclient.get_descendant_regions(region_id, REGION_LEVELS['district'])</code></p><p> </p><p>instead of <code>gclient.get_descendant_regions(region_id, 5)  # 5 is district</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>sure makes sense</p>